webpack/assets/javascripts/react_app/components/LoginPage/helpers.js

```diff
@@ -26,7 +27,7 @@ export const adjustAlerts = alerts => {
 
 export const defaultFormProps = {
   attributes: {
-    action: '/users/login',
+    action: foremanUrl('/users/login'),
```

Comment:
Should this use `urlBuilder`
```suggestion
    action: urlBuilder('users', 'login'),
```

Reply:
would look better for sure :+1: